docs/source/notebooks/api_quickstart.ipynb

```diff
@@ -16,12 +16,13 @@
      "name": "stdout",
```

Comment:
<p>As per our discussion, this section whole should probably be replaced by something like:</p><blockquote>It's worth highlighting the design choice we made with&nbsp;<code>logp</code>.&nbsp;As you can see above,&nbsp;<code>logp</code>&nbsp;is being called with arguments, so it's a method of the model instance. More precisely, it puts together a function based on the current state of the model -- or on the state given as argument to&nbsp;<code>logp</code>&nbsp;(see example below).</blockquote><blockquote>For diverse reasons, we assume that a&nbsp;<code>Model</code>&nbsp;instance isn't static. If you need to use&nbsp;<code>logp</code>&nbsp;in an inner loop and it needs to be static, simply use something like&nbsp;<code>logp = model.logp</code>. Here is an example below -- note the caching effect and the speed up:</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>